### PR TITLE
Rename hatch to release-hatch

### PR DESCRIPTION
**Pull request by madwort**
* for consistency!

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
     - name: Set up LXD
       run: |
-        sudo snap set lxd shiftfs.enable=true
+        sudo snap set lxd shiftfs.enable=true | { sleep 20 && sudo snap set lxd shiftfs.enable=true; }
```
Comment (bloodearnest) [inline .github/workflows/test.yaml:15]: typo s/|/||/?

The autorefresh blocker seems to have done now anyway?

Comment (madwort) [reply .github/workflows/test.yaml]: 😱

Comment (bloodearnest) [reply .github/workflows/test.yaml]: Hah, and now I've committed to fix that, I can't approve!

**Review by bloodearnest — APPROVED** [review]

File: .github/workflows/test.yaml (hunk 1 above)
Comment (bloodearnest) [reply .github/workflows/test.yaml]: oh no, i can.merging!

**Pull request merged**